test/e2e/network_segmentation.go

```diff
@@ -33,9 +33,12 @@ import (
 )
 
 const openDefaultPortsAnnotation = "k8s.ovn.org/open-default-ports"
+const RequiredUDNNamespaceLabel = "k8s.ovn.org/primary-user-defined-network"
 
 var _ = Describe("Network Segmentation", func() {
 	f := wrappedTestFramework("network-segmentation")
+	// disable automatic namespace creation, we need to add the required UDN label
+	f.SkipNamespaceCreation = true
```

Comment:
ohh nice
self-reminder: this is going to screw up my admin network policy e2e tests which come from upstream k8s-sig
@trozet given we can't have external entities create namespaces anymore, is there a way for us to actually port the upstream k8s services and netpol tests like we planned to run them all on UDNs? I assume not...